ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -71,6 +71,31 @@ tests:
       env:
       - default: /usr/local/ci-secrets/cincinnati-secrets-cincinnati-credentials/registry_credentials_docker.json
         name: CINCINNATI_TEST_CREDENTIALS_PATH
+      - default: /usr/local/ci-secrets/ci-image-sa-dockercfg/.dockercfg
+        name: CINCINNATI_CI_DOCKERCFG_PATH
+      - default: /usr/local/ci-secrets/public-ci-image-sa-dockercfg/.dockercfg
+        name: CINCINNATI_CI_PUBLIC_DOCKERCFG_PATH
+      - default: ${SHARED_DIR}/ci-image-sa-dockerjson
```

Comment:
Export filename here and add `${SHARED_DIR}` in `commands`?

Reply:
Addressed, thanks!